open_spiel/python/mfg/algorithms/distribution.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, game, policy, root_state=None):
     """
     super(DistributionPolicy, self).__init__(game)
     self._policy = policy
+    self.game = game
```

Comment:
Not needed, the parent class (distribution.Distribution) should already have this field.

Reply:
Thanks